packages/Card/src/components/Title/Title.js

```diff
@@ -5,17 +5,22 @@ import * as sc from "./Title.styles";
 const propTypes = {
   /** Body content of the Title. */
   children: PropTypes.node,
+  /** Semantic level of Heading for the title */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
 };
 
 const defaultProps = {
   children: null,
+  level: 1,
```

Comment:
`2` would probably be a better default.  It is better practice to use the `h1` for the top level page title.